cobigen-maven/cobigen-maven-systemtest/src/test/java/com/devonfw/cobigen/maven/systemtest/Devon4JTemplateTest.java

```diff
@@ -0,0 +1,167 @@
+package com.devonfw.cobigen.maven.systemtest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Collections;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.shared.invoker.DefaultInvocationRequest;
+import org.apache.maven.shared.invoker.DefaultInvoker;
+import org.apache.maven.shared.invoker.InvocationRequest;
+import org.apache.maven.shared.invoker.InvocationResult;
+import org.apache.maven.shared.invoker.Invoker;
+import org.junit.Test;
+
+import com.devonfw.cobigen.maven.systemtest.config.constant.MavenMetadata;
+import com.devonfw.cobigen.maven.test.AbstractMavenTest;
+
+/**
+ *
+ */
+public class Devon4JTemplateTest extends AbstractMavenTest {
```

Comment:
Please cleanup the devon4j tests as there is a lot unneeded template code.
As you can see in templates-oasp4j, there is just one template at all, but here we have quite a lot, which is needless and makes the tests less clear.

Reply:
cleaned up to use a single ETO file with a to increment instead.